src/Illuminate/Testing/PendingCommand.php

```diff
@@ -192,6 +193,26 @@ public function assertExitCode($exitCode)
         return $this;
     }
 
+    /**
+     * Assert that the command has the success exit code.
+     *
+     * @return $this
+     */
+    public function assertSuccessful()
+    {
+        return $this->assertExitCode(Command::SUCCESS);
+    }
+
+    /**
+     * Assert that the command has the failure exit code.
+     *
+     * @return $this
+     */
+    public function assertFailed()
```

Comment:
Wonder if this should assert exit code not equal to zero. Commands can fail with exit codes other than 1.

Reply:
You're totally right, @GrahamCampbell, I updated the implementation based on this 👍